core/app/models/spree/import.rb

```
@@ -0,0 +1,44 @@
+module Spree
+  class Import < Spree.base_class
+    ALLOWED_CONTENT_TYPES = %w[text/csv].freeze
+    # Associations
+    
```

RuboCop (1.75.5)

[convention] Trailing whitespace detected.

(Layout/TrailingWhitespace)